controllers/secretgenerator/secretgenerator_controller.go

```diff
@@ -131,47 +133,40 @@ func (r *SecretGeneratorReconciler) Reconcile(ctx context.Context, request ctrl.
 		},
 	}
 
-	generatedSecretKey := types.NamespacedName{
-		Name: generatedSecret.Name, Namespace: generatedSecret.Namespace,
-	}
-	err = r.Client.Get(ctx, generatedSecretKey, generatedSecret)
+	err = r.Client.Get(ctx, client.ObjectKey{Name: generatedSecret.Name, Namespace: generatedSecret.Namespace}, generatedSecret)
```

Comment:
What does the `types.NamespacedName` change to `client.ObjectKey` achieve?

Reply:
no real difference.
both `generatedSecretKey` and `client.ObjectKey{}` return` types.NamespacedName` type,
i went  for the later one only because of inline style than introducing a one time variable

Reply:
FWIW, there is also a function helper https://github.com/kubernetes-sigs/controller-runtime/blob/b901db121e1f53c47ec9f9683fad90a546688c3e/pkg/client/interfaces.go#L35

Reply:
nice, then it can be simplified to `err = r.Client.Get(ctx, client.ObjectKeyFromObject(generatedSecret), generatedSecret)`